src/client/src/app/modules/offline/components/content-manager-info-pop-up/content-manager-info-pop-up.component.html

```diff
@@ -5,9 +5,12 @@
   </div>
 
   <div class="sb-modal-content flex-dc">
-    <p class="text-center fmedium import-content">
+    <p class="text-center fmedium import-content" *ngIf="!isChangeLocation">
       {{resourceService?.frmelmnts?.lbl?.desktop?.failedListLabel}}
     </p>
+    <p class="text-center fmedium import-content" *ngIf="isChangeLocation">
```

Comment:
user suggest drive is this but this should be select box user can select other drives other than drives which are full